src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationServiceTest.java

```diff
@@ -20,6 +22,7 @@
 @RunWith(MockitoJUnitRunner.class)
 public class DefendantResponseNotificationServiceTest extends BaseNotificationServiceTest {
 
+    private final Claim claim = SampleClaim.getWithDefaultResponse();
```

Comment:
What response is default? Would be nice to know whether that is part admission, full defence or something else. Please name method accordingly.

Reply:
The response is a SampleFullDefenceResponse, so I think renaming to:

SampleClaim.getWithValidFullDefence();